cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -186,29 +198,6 @@ class OrtMainFunTest : StringSpec() {
         }
 
         "Analyzer creates correct output" {
-            val expectedResult = patchExpectedResult(
```

Comment:
Why was this test removed?

Reply:
I have added an explanation to the commit message.